src/Adapter/Builder/GridFSAdapterDefinitionBuilder.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the flysystem-bundle project.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace League\FlysystemBundle\Adapter\Builder;
+
+use Doctrine\ODM\MongoDB\DocumentManager;
+use League\Flysystem\GridFS\GridFSAdapter;
+use MongoDB\Client;
+use MongoDB\GridFS\Bucket;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Jérôme Tamarelle <jerome@tamarelle.net>
+ *
+ * @internal
+ */
+class GridFSAdapterDefinitionBuilder extends AbstractAdapterDefinitionBuilder
```

Comment:
final?

Reply:
Other similar classes are not final, but that may be missing. Since this classes are internal, we could make them all final. I do prefer having a separate PR for all the classes.